### PR TITLE
feat(web): Use bsc info gateway

### PR DESCRIPTION
**Pull request by chef-jojo**
---

## PR-Codex overview
This PR updates various GraphQL queries and constants related to stable swap information retrieval.

### Detailed summary
- Added `as const` for subgraphs type in `@pancakeswap/chains`
- Updated GraphQL queries in multiple files to use `ID` instead of `String`
- Modified orderBy fields in queries to use `ETH` instead of specific tokens
- Updated import paths and removed unnecessary imports

> The following files were skipped due to too many changes: `apps/web/src/config/constants/endpoints.ts`, `apps/web/src/state/swap/fetch/fetchDerivedPriceData.ts`, `scripts/updateLPsAPR.ts`, `apps/web/src/state/info/queries/tokens/priceData.ts`, `apps/web/src/utils/graphql.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/config/constants/endpoints.ts [hunk 1]
```diff
@@ -45,13 +44,26 @@ export const ACCESS_RISK_API = 'https://red.alert.pancakeswap.com/red-api'
 
 export const CELER_API = 'https://api.celerscan.com/scan'
 
-export const INFO_CLIENT_WITH_CHAIN = V2_SUBGRAPHS
+export const EXPLORER_API = 'https://info-gateway.pancakeswap.com'
```
Comment (chefjackson) [inline apps/web/src/config/constants/endpoints.ts:47]: do we need to move to the pkg as well?

Comment (chef-jojo) [reply apps/web/src/config/constants/endpoints.ts:47]: This will be only used for web. For pkg by default uses the public subgraph

Comment (chefjackson) [reply apps/web/src/config/constants/endpoints.ts:47]: ok but public subgraph is gonna be deprecated right?

Comment (chef-jojo) [reply apps/web/src/config/constants/endpoints.ts:47]: No

**Review by chefjackson — APPROVED** [review]

**Pull request merged**